pkg/crc/preflight/preflight_linux.go

```diff
@@ -114,6 +114,14 @@ var vsockPreflightChecks = Check{
 	cleanup:            removeVsockCrcSettings,
 }
 
+var wsl2PreflightChecks = Check{
```

Comment:
Instead of separate out this test shouldn't this be part of generic one because it is going to depend on any linux distro which is running as part of WSL2 right?